analysisCode/src/include/BreitFrame.h

```diff
@@ -0,0 +1,8 @@
+#ifndef BREITFRAME_H
+#define BREITFRAME_H
+
+#include <eicsmear/erhic/EventPythia.h>
+ 
```

Comment:
Let's just go ahead and make this a class (albeit a small one) just to be consistent with the other files in this project

Reply:
Will do, if there are any other functionalities you would like me to add, just let me know. I could add a function to boost particles back to the lab frame, but I am not convinced this is necessary.

Reply:
Yeah I think you are right, if we want to do jet finding the lab frame we can just use the particles as pythia/eic smear spit them out.